app/views/shared/_progress_bar.html.slim

```diff
@@ -0,0 +1,16 @@
+.govuk-grid-row
+  ol.progress-bar--container.govuk-list role="progressbar"
+    - array_of_colours = bar.milestones.map {|item| bar.icon(item)[2]}
+    - last_green = array_of_colours.rindex(:green)
+
+    - bar.milestones.each_with_index.map do |item, index|
```

Comment:
@agoldstone93 let's attempt to shift logic into the decorator so it's not in the template. It'll make a unit test for the decorator easier to achieve too.